app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
```

Comment:
- [ ] As part of this module requirement, _User_ model should have a method that returns **3 most recent posts**. This should be similar to:
```ruby
# app/models/user.rb
...

def recent_post
  posts.order('created_at Desc').limit(3)
end
```